code/game/objects/items/implants/implant_abductor.dm

```diff
@@ -13,22 +13,46 @@
 		to_chat(imp_in, span_warning("You must wait [timeleft(on_cooldown)*0.1] seconds to use [src] again!"))
 		return
 
-	home.Retrieve(imp_in,1)
+	if(!home)
+		if(!link_pad())
+			imp_in.balloon_alert(imp_in, "no teleport pads detected!")
+			return
```

Comment:
```suggestion
	if(isnull(home) && !link_pad())
			imp_in.balloon_alert(imp_in, "no teleport pads detected!")
			return
```

Is this what you're trying to do? Should work for what you're trying for. BTW `isnull()` looks cleaner for stuff like this (and is faster too to some degree, but it's just stylistically better to use isnull on typed vars in contexts like these (using `!` is fine for booleans))